qemu/tests/hv_enforce_cpuid_msr_check.py

```diff
@@ -0,0 +1,90 @@
+from virttest import error_context
+from provider import qemu_img_utils
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Add 'hv_enforce_cpuid' option to try to use a forbidden feature.
+    1. Boot the guest with following hv flags.
+    eg: -cpu host,hv-relaxed,hv-enforce-cpuid
+    2. Run command rdmsr 0x40000002 in guest.
+    3. Check the results.
+    4. Boot the guest without 'hv-enforce-cpuid'.
+    5. Run command rdmsr 0x40000002 in guest.
+    6. Check the results.
+
+    param test: the test object
+    param params: the test params
+    param env: the test env object
+    """
+
+    def _set_env(session):
+        """
+        Set up the system environment before test
+
+        param session: the session of vm
+        """
+        # modify the subscription-manager
+        session.cmd_output_safe(params.get("modify_submanager_cmd"))
+        # get the system version
+        ver = session.cmd_output_safe(params.get("system_version_cmd"))
+        ver = ver.split("\n")[0]
+        # install repo
+        session.cmd_output_safe(params.get("repo_install_cmd").replace("${X_VER}", ver))
+        # install msr-tools
+        if session.cmd_status(params.get("msr_install_cmd")):
+            test.error("Failed to install msr-tools")
+
+    def _clean_env(session):
+        """
+        Clean the system environment after test
+
+        param session: the session of vm
+        """
+        # restore the subscription-manager
+        session.cmd_output_safe(params.get("restore_submanager_cmd"))
+
+    def _run_msr_tools(session):
+        """
+        run msr-tools cmd
+
+        params session: the session of vm
+        return res: the result of output
+        """
+        session.cmd_output_safe(params.get("modprobe_cmd"), timeout=360)
+        res = session.cmd_output_safe(params.get("rdmsr_cmd"), timeout=360)
+        return res
+
+    error_context.context("The case starts...", test.log.info)
+    error_context.context("The case of cpu with hv flags: ", test.log.info)
```

Comment:
better be:   
error_context.context("Boot the guest with 'hv-enforce-cpuid ", test.log.info)